recipe/meta.yaml

```diff
@@ -31,29 +30,34 @@ requirements:
     - wheel
     - llvmdev 11.1.*
     - llvm 11.1.*
-    - zlib
-    - vs2015_runtime  # [win]
+    - zlib 1.2.*
+    - vs2015_runtime 14.16.27012 # [win]
   run:
     - python
     - zlib
     - vs2015_runtime  # [win]
 
 test:
+  requires:
+    - pip
   imports:
     - llvmlite
     - llvmlite.binding
   commands:
+    - pip check
     - export LLVMLITE_DIST_TEST=''  # [unix]
     - set LLVMLITE_DIST_TEST=""     # [win]
     - python -m llvmlite.tests  # [not osx]
     - python -m llvmlite.tests || true  # [osx]
```

Comment:
Have you tried the osx tests without || true?

Reply:
nope , will try